.github/actions/archive-paparazzi-reports/action.yml

```diff
@@ -0,0 +1,16 @@
+name: '📦 Archive Paparazzi reports'
+description: 'Archive Paparazzi reports and deltas'
+runs:
+  using: composite
+  steps:
+    - name: 'Archive report'
+      uses: actions/upload-artifact@v3
+      with:
+        name: paparazzi-html
+        path: '**/build/reports/paparazzi'
+    - name: 'Archive deltas'
+      if: ${{ hashFiles('**/out/failures/**.png') != '' }}
+      uses: actions/upload-artifact@v3
+      with:
+        name: paparazzi-delta
+        path: '**/out/failures/**.png'
```

Comment:
SHouldn't this be '**/out/failures/delta-**.png' or do we want to upload all screenshots?

Reply:
I suppose we want both. `Delta-*` are composited images that could differ from the real content.

Reply:
Actually they have the previous one on the left side | the delta in the middle | the new one on the right side so imo we can just use these

Reply:
Yes I know they combine both images, but the resulting composition does not correspond exactly to what as been produced by paparazzi. So I prefer we keep both files for now.

Reply:
Good to me